app/src/main/java/com/codesoom/models/JsonConverter.java

```diff
@@ -0,0 +1,55 @@
+package com.codesoom.models;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Map;
+
+/**
+ * Task > Json | Json > Task 변환클래스
+ */
+public class JsonConverter {
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    /**
+     * Json > Task 객체로 매핑
+     * @param content - Task 객체에 들어갈 데이터
+     * @return Task - 매핑해준 Task 객체 반환
+     * @throws JsonProcessingException
+     */
+    public Task toTask(String content) throws JsonProcessingException {
+        return mapper.readValue(content,Task.class);
+    }
```

Comment:
잘 작성하셨지만 저를 따라서 다음과 같이 수정해 보세요. 그리고 다른 주석들도 수정해 보세요.
```suggestion
    /**
     * 주어진 Json 형식의 문자열을 읽고 할 일 객체로 변환해 리턴합니다.
     * @param content json 형식의 문자열
     * @return Task 변환된 할 일 객체
     * @throws JsonProcessingException json 문자열 변환 중 문제가 발생했을 경우
     */
    public Task toTask(String content) throws JsonProcessingException {
        return mapper.readValue(content,Task.class);
    }
```